src/API/Site/Controllers/RestAPI/SyncController.php

```diff
@@ -259,7 +227,7 @@ protected function get_request_params( Request $request ): array {
 			return [];
 		}
 
-		$params       = array_intersect_key( $request_params, self::DEFAULT_SYNC_MODE );
+		$params       = array_intersect_key( $request_params, self::get_default_sync_mode() );
```

Comment:
We haven't made this function static, so isn't it better to call `$this->get_default_sync_mode`?

Reply:
Forgot to change that after refactoring.